.travis.yml

```diff
@@ -4,5 +4,27 @@ node_js:
 cache:
   directories:
   - node_modules
+
+
+addons:
+  code_climate
+
```

Comment:
Remove the unnecessary spaces in this file

Reply:
Thank for the feedback, this has been done.